packages/request-logic/src/config.ts

```diff
@@ -1,6 +1,6 @@
 export default {
   specificationVersion: {
-    current: '2.0.2',
+    current: '2.0.3',
```

Comment:
Why are we bumping the version? Is adding this new identity making things not retrocompatible? 
If so, should we be checking the version when checking for ETHEREUM_SMART_CONTRACT type?